### PR TITLE
Explicit RK Oliver (6 stages, 5th order) for start…

### PR DESCRIPTION
**Pull request by frankschae**
…ing singularity #864. I am very unsure whether I have set correctly integrator.destats.nf += 6 and integrator.kshortsize = 6.. Unfortunately, the tests fail (tolerance must be significantly increased) -- it would be great if someone could have a quick look and point me to possible problems :)  I have written out the Butcher table. I could push it to DivEqDevTools if that helps (?)

### CODE REVIEW

File: test/algconvergence/ode_convergence_tests.jl [hunk 1]
```diff
@@ -33,6 +33,9 @@ testTol = 0.2
   sim3 = test_convergence(dts2,prob,KuttaPRK2p5(threading=false))
   @test sim3.𝒪est[:l∞] ≈ 5 atol=testTol
 
+  sim3 = test_convergence(dts2,prob,RKO65())
```
Comment (ChrisRackauckas) [inline test/algconvergence/ode_convergence_tests.jl]: what does `plot(sim3)` look like?

File: test/algconvergence/ode_convergence_tests.jl [hunk 2]
```diff
@@ -33,6 +33,9 @@ testTol = 0.2
...
+  @test sim3.𝒪est[:l∞] ≈ 5 atol=testTol
```
Comment (ChrisRackauckas) [inline test/algconvergence/ode_convergence_tests.jl]: what result are you getting here?

Comment (frankschae) [reply test/algconvergence/ode_convergence_tests.jl]: Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 0.9645038959802423 ≈ 5 (atol=0.2)

Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 0.9645038959807267 ≈ 5 (atol=0.2)

File: test/algconvergence/ode_convergence_tests.jl (hunk 1 above)
Comment (frankschae) [reply test/algconvergence/ode_convergence_tests.jl]: Foe the 2D inlace: 

prob_ode_2Dlinear

Comment (frankschae) [reply test/algconvergence/ode_convergence_tests.jl]: For the 1D out of place

prob_ode_linear

**Comment by ChrisRackauckas** [comment]
Looks like perfect first order convergence. This means there's a coefficients that's incorrect somewhere. Does each row of the A tableau sum to the corresponding c_i?

**Comment by frankschae** [comment]
Do you mean A*e =c? They explicitly say that this condition is dropped in Section II.

**Comment by ChrisRackauckas** [comment]
I don't have access to the paper, but how do they get higher convergence without it?

**Comment by frankschae** [comment]
it holds only for indices greater than 1.. so they set c1 != 0  while at the same time they set b1 =0. But I just spotted that indeed also c6 != sum[A[6, I]] if I sum up the values in the table..:
1 != 0 + 123/300 + 5/2 + 3/20 - 75/38 = 516/475

File: test/algconvergence/ode_convergence_tests.jl (hunk 2 above)
Comment (frankschae) [reply test/algconvergence/ode_convergence_tests.jl]: With the updated value I get: 

Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 4.9872356619789535 ≈ 5 (atol=0.2)


Expression: ≈(sim3.𝒪est[:l∞], 5, atol=testTol)
   Evaluated: 4.998263889697244 ≈ 5 (atol=0.2)

**Comment by ChrisRackauckas** [comment]
Thanks!

**Pull request merged**